src/misc/unicode.cpp

```diff
@@ -1295,16 +1821,13 @@ static bool construct_mapping(const uint16_t code_page)
 	}
 	already_tried.insert(code_page);
 
-	assert(config_mappings.count(code_page));
-	assert(!mappings_normalized_by_codepage.count(code_page));
-	assert(!mappings_decomposed_by_codepage.count(code_page));
-	assert(!mappings_reverse_by_codepage.count(code_page));
+	assert(!per_code_page_mappings.count(code_page));
```

Comment:
Suggest quantity comparison (count == 0)

Reply:
OK, changed - everywhere in this file.